rust/operator-binary/src/crd/mod.rs

```diff
@@ -130,6 +132,10 @@ pub mod versioned {
 
         #[fragment_attrs(serde(default))]
         pub resources: Resources<StorageConfig>,
+
+        /// This field controls which [ListenerClass](https://docs.stackable.tech/home/nightly/listener-operator/listenerclass.html) is used to expose the webserver.
```

Comment:
"webserver" is the wrong term for OpenSearch. In the documentation, it is just called "HTTP communication".

https://docs.opensearch.org/latest/install-and-configure/configuring-opensearch/network-settings/

```suggestion
        /// This field controls which [ListenerClass](https://docs.stackable.tech/home/nightly/listener-operator/listenerclass.html) is used to expose the HTTP communication.
```

Reply:
Thanks - that was probably a C&P error